grabbit/src/main/resources/META-INF/spring/client-batch-job.xml

```diff
@@ -32,16 +32,35 @@
     -->
 
     <import resource="client-batch-job-context.xml"/>
+
+    <import resource="client-namespace-step.xml" />
     <import resource="client-jcrNodes-step.xml" />
-    <import resource="client-preprocessor-step.xml" />
+    <import resource="client-deleteBeforeWrite-steps.xml" />
 
     <batch:job id="clientJob" xmlns="http://www.springframework.org/schema/batch" job-repository="clientJobRepository">
-        <batch:step id="clientPreprocessor" next="clientWorkflowOff">
-            <batch:tasklet ref="clientPreprocessTasklet" transaction-manager="clientTransactionManager"/>
-        </batch:step>
-        <batch:step id="clientWorkflowOff" next="clientJcrNodes">
-            <batch:tasklet ref="clientWorkflowOffTasklet" transaction-manager="clientTransactionManager"/>
+        <!--  Do any presync tasks concurrently that we can, e.g turning off workflows -->
+        <batch:split id="preSyncFlow" task-executor="asyncTaskExecutor" next="clientNamespaceSync">
+            <batch:flow>
+                <batch:step id="clientWorkflowOff">
```

Comment:
`clientWofkflowOff` is VERY lightweight step. Having said that, even if it is light weight, it actually does manage concurrency internally (so there is already multi-threading handling involved in the actual step and on that we are spawning different thread for the step it self).

Do we know this split of steps is going to give us tangible benefits?

If not, then I vote we keep it simple and have the steps be sequential. If there are no tangible benefits, I think this way will be harder to maintain (as you know we already have 1 Thread Per Path in the config)

cc: @jdigger